md-dao/src/main/java/com/wipro/ats/bdre/md/dao/ProcessDependencyDAO.java

```diff
@@ -42,6 +42,7 @@
     private static final Logger LOGGER = Logger.getLogger(ProcessDependencyDAO.class);
     @Autowired
     SessionFactory sessionFactory;
+    private static final String DELETEFLAG="deleteFlag";
 
     public List<ProcessDependencyInfo> listUD(Integer parentProcessId) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "listUD" is 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)